packages/next/src/server/config-shared.ts

```diff
@@ -289,6 +282,13 @@ export interface ExperimentalConfig {
 
   webVitalsAttribution?: Array<(typeof WEB_VITALS)[number]>
 
+  /**
+   * A list of extensions to resolve when importing files.
+   *
+   * @see [Resolve Extensions](https://nextjs.org/docs/app/api-reference/next-config-js/turbo#resolve-extensions)
+   */
+  resolveExtensions?: string[]
```

Comment:
We should list the default here and in the docs. Right now it looks like none are used by default